.github/workflows/teleport-create-user.yaml

```diff
@@ -51,17 +61,19 @@ jobs:
             expect "Enter password for Teleport user ${{ inputs.user_manage }}:"
             send "${{ inputs.password_user_manage }}\r"
             expect "Enter an OTP code from a device:"
-            send "${{ inputs.OTP-admin }}\r"
+            send "${{ inputs.OTP_secret }}\r"
             expect eof
           EOF
       - name: Verify login success
         run: tctl status 
       - name: Create user name
-        run: tctl users add ${{ inputs.user_name }} --roles=${{ inputs.user_role }}
-        id: user_name_info  
+        id: user_info
+        run: |
+          tctl users add ${{ inputs.user_name }} --roles=${{ inputs.user_role }} >> tmp.txt
+          echo "::set-output name=USER_INFO::$(cat tmp.txt | grep http)"
       - name: Slack Notification
         uses: rtCamp/action-slack-notify@v2
         env:
-          SLACK_MESSAGE: ${{join(steps.user_name_info.outputs.*, '\n')}}
-          SLACK_TITLE: Send username info
+          SLACK_MESSAGE: ${{ steps.user_info.outputs.USER_INFO }}
+          SLACK_TITLE: User `${{ inputs.user_name }}` has been created access URL to complete user setup
```

Comment:
has been created.
Access URL to complete user setup: 
(there must be a full stop between the two sentences to make it clearer)